open-api/rest-catalog-open-api.yaml

```diff
@@ -2731,6 +2967,32 @@ components:
           additionalProperties:
             type: string
 
+    PrePlanTableResult:
+      description:
+        When doing prePlan the result is a list of plan-tasks.
+      type: object
+      required:
+        - plan-tasks
+      properties:
+        expiration-time-ms:
```

Comment:
I don't think this is needed, it can be a part of each plan task's opaque object content